apps/metabase/app.yaml

```diff
@@ -0,0 +1,83 @@
+{
+    "id": "/metabase",
+    "cmd": null,
+    "cpus": 2,
+    "mem": 2048,
+    "disk": 0,
+    "gpus": 0,
+    "instances": 1,
+    "executor": "",
+    "requirePorts": false,
+    "constraints": [
+        [
+            "secrets",
+            "LIKE",
+            "true"
+        ],
+    ],
+    "container": {
+        "type": "DOCKER",
+        "volumes": [
+            {
+                "containerPath": "/etc/ocf-metabase",
+                "hostPath": "/opt/share/docker/secrets/metabase",
+                "mode": "RO"
+            }
+        ],
+        "docker": {
+            "image": "docker.ocf.berkeley.edu/metabase",
+            "network": "BRIDGE",
+            "portMappings": [
+                {
+                    "containerPort": 3000,
+                    "hostPort": 0,
+                    "servicePort": 10010,
+                    "protocol": "tcp",
+                    "name": "main",
+                    "labels": {}
+                }
+            ],
+            "privileged": false,
+            "parameters": [],
+            "forcePullImage": false
+        }
+    },
+    "env": {
+        "HOST": "",
+        "PORT": ""
+    },
+    "healthChecks": [
+        {
+            "path": "/",
```

Comment:
You mentioned before that the health check caused problems because it was slow to start up. Will that be a problem at all? What made it faster to start up later?

Reply:
The healthchecks on the very first time I started it were a problem because it needed to provision the database and that was really slow, but Marathon was killing it and restarting it before it could finish initial setup. To fix that I manually started a container with the prod creds, finished setting up the database, and then started the marathon task again, which now works fine.